packages/core-app-api/src/apis/implementations/auth/onelogin/OneLoginAuth.ts

```diff
@@ -76,7 +76,7 @@ export default class OneLoginAuth {
       oauthRequestApi,
       provider,
       environment,
-      defaultScopes: ['openid', 'email', 'profile', 'offline_access'],
+      defaultScopes: ['openid', 'email', 'profile'],
```

Comment:
I could not test this. If this defaults are working I assume I can remove the override in the file `plugins/auth-backend-module-onelogin-provider/src/authenticator.ts`

Reply:
This scope lets us get a refresh token. Since I do not use this provider, I'm honestly not sure whether this is OK or not. The price to pay is that refresh tokens won't be used, I presume, rather than causing larger breakages.

@Rugvip do you know more about this? Otherwise let's ship with this change and then adjust if needed in a follow-up.

Reply:
We can leave it if we need, but I am sure that it will not work in the start method. So we need to keep the other one. And we can leave this with offline_access

Reply:
Let's aim to not make any functional changes during the migration. Ideally we lift over the logic to the module exactly as is and release that in the next mainline in two week. Then we can ship these somewhat more risky changes after that so it has a couple of next release for people to try it out.

Btw, please remove the existing implementation of the onelogin provider in the auth-backend and replace it with a similar shim to what the other migrated providers use.